projects/plugins/jetpack/modules/likes.php

```diff
@@ -367,10 +381,10 @@ function load_admin_css() {
 	}
 
 	/**
-	* Load the JS required for loading the like counts.
-	*/
-	function enqueue_admin_scripts() {
-		if ( empty( $_GET['post_type'] ) || 'post' == $_GET['post_type'] || 'page' == $_GET['post_type'] ) {
+	 * Load the JS required for loading the like counts.
+	 */
+	public function enqueue_admin_scripts() {
+		if ( empty( $_GET['post_type'] ) || 'post' === $_GET['post_type'] || 'page' === $_GET['post_type'] ) {
```

Comment:
This line here needs nonce verification and is the last thing that needs addressing, but it doesn't look like a nonce is set and I'm honestly not sure where in this case we'd set one.

Reply:
Hmm. It's not really *processing* form data anyway. It's just adding some scripts. I might just ignore this one. But you might want to ask someone more familiar with WP code than I am for confirmation.

Reply:
Yeah, I'll cc @kraftbj here for his expert opinion.

Reply:
Apologies for the delay, but agreed on ignoring.